config/device-classes/generic/vmware-esxi.yaml

```
@@ -0,0 +1,67 @@
+name: "vmware-esxi"
+
+config:
+  components:
+    cpu: true
+    memory: true
+    disk: true
+    server: true
+
+match:
+  logical_operator: OR
+  conditions:
+    - type: SysObjectID
+      match_mode: startsWith
+      values:
+        - ".1.3.6.1.4.1.6876.4.1"
+    - type: SysDescription
+      match_mode: regex
+      values:
+        - '^VMware ESXi'
+
+identify:
+  properties:
+    vendor:
+      - detection: constant
+        value: "VMware"
+    model:
+      - detection: SysDescription
+    serial_number:
+      - detection: snmpget
+        oid: .1.3.6.1.2.1.47.1.1.1.1.11.1
+    os_version:
+      - detection: snmpget
+        oid: .1.3.6.1.4.1.6876.1.1-7
```

invalid character `-` in oid
```suggestion
        oid: .1.3.6.1.4.1.6876.1.2.0
```